luarules/gadgets/game_micro_wars.lua

```diff
@@ -0,0 +1,1011 @@
+-- game_micro_wars
+
+if not gadgetHandler:IsSyncedCode() then
+    return
+end
+
+-- Load mod options
+local modOptions = Spring.GetModOptions() or {}
+
+local microWarsEnabled = modOptions.micro_wars_enabled or false
+if not microWarsEnabled then
+    return false -- Disable this gadget if Micro Wars is not enabled
+end
+
+local roundTime = (modOptions.round_time or 5) * 60 * 30 -- Convert minutes to game frames (30 frames per second)
+local numberOfControlPoints = modOptions.number_of_control_points or 10
+local productionMode = modOptions.production_mode or false
+local maxRoundsMode = modOptions.max_rounds_mode or false
+local maxNumberOfRounds = modOptions.max_number_of_rounds or 10
+local controlPointUnitConversion = modOptions.control_point_unit_conversion or 1
+local allowRoundResign = modOptions.allow_round_resign or false
+local battlefieldMode = modOptions.micro_wars_battlefield_mode or true
+local endRoundEarlyPercentage = modOptions.end_round_early_percentage or 50
+
+-- Correct initialization for despawnUnits
+local despawnUnits = true -- Default value
+if modOptions.micro_wars_despawn == nil then
+    despawnUnits = true
+else
+    despawnUnits = modOptions.micro_wars_despawn
+end
+
+local teams = Spring.GetTeamList()
+local gaiaTeamID = Spring.GetGaiaTeamID()
+
+-- Custom unit spawn configuration per round
+local unitSpawnConfigs = {
+    ["Basic T1 - T3"] = {  -- Already provided in your original script
+        [1] = {
+			{unitName = "armpw", count = 20},    -- Pawns
+			{unitName = "armpwt4", count = 10},  -- Rocket Bots
```

Comment:
these dont look like rocket bots

Reply:
Whoops, this is correct. I swapped in the epic pawns to test that scavenger units were working. This doesn't change any mechanics and the battle still plays fine. 
In general, the army compositions are rather rough and will need some human multiplayer testing to ultimately figure out what's the most fun. For this specific check, what should I do?

Reply:
I think you should ignore it and keep things as-is, since it will resolve itself anyway when internal unit names stop being stupid.